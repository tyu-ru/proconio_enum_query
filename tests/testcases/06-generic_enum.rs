#[proconio_enum_query::derive_query]
#[allow(dead_code)]
#[derive(PartialEq, Debug)]
enum Query<T> {
    A(T),
    B,
}

fn main() {}
