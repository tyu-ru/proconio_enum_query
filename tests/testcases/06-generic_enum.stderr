error: generic enum not supported.
 --> $DIR/06-generic_enum.rs:4:11
  |
4 | enum Query<T> {
  |           ^^^
